### PR TITLE
Adding PHP Client draft

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
@mikethebeer As new owner of the PHP libraries, are you happy with this?

This is not the in-depth documentation, rather the 'getting started' overview of how to add a library to your project and connect to Crate.

### CODE REVIEW

File: pages/docs/getting-started/clients/php.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+```json
+{
+  "require": {
+    "crate/crate-pdo":"~0.3.0"
+  }
+}
+```
+
+And installing with `composer intall`.
```
Comment (mikethebeer) [inline pages/docs/getting-started/clients/php.html]: ``s/intall/install/g``

Comment (ChrisChinchilla) [inline pages/docs/getting-started/clients/php.html]: @mikethebeer They all kind of work, but you're right :) Will fix up.

File: pages/docs/getting-started/local/linux.html [hunk 2]
```diff
@@ -190,7 +190,15 @@
 sudo apt-get install crate
 ```
 
-#### Testing and Unstable Releases
+### Start Crate
+Crate should have been started automatically by upstart and can be stopped or restarted with:
+
+```bash
+sudo service crate stop
+sudo service crate restart
```
Comment (chaudum) [inline pages/docs/getting-started/local/linux.html]: `restart` will stop and start the service

Comment (ChrisChinchilla) [inline pages/docs/getting-started/local/linux.html]: @chaudum Yes, that's what the line says? I'm not sure what your point is?

**Pull request merged**